### PR TITLE
Added new logo suite component

### PR DESCRIPTION
**Pull request by rezrah**
## Summary

New logo suite component for displaying vendor or product logos.

🔗  [Interface guidelines](https://primer-3e1a8f5af5-26139705.drafts.github.io/components/LogoSuite)
🔗  [React documentation](https://primer-3e1a8f5af5-26139705.drafts.github.io/components/LogoSuite/react)
🔗  [Storybook examples](https://primer-3e1a8f5af5-26139705.drafts.github.io/storybook/?path=/story/components-logosuite--default)

## PR readiness 

Please feel free to leave feedback and review this PR, even if all items aren't checked off below. 

- [x] base component
- [x] unit tests
- [x] design review
- [x] visual tests
- [x] storybook
- [x] interface guidelines
- [x] react documentation

## What should reviewers focus on?

- Reviewing documentation
- Reviewing API
- Code review
- Timing for marquee effect

## Contributor checklist:

- [x] All new and existing CI checks pass
- [x] Tests prove that the feature works and covers both happy and unhappy paths
- [x] Any drop in coverage, breaking changes or regressions have been documented above
- [x] New visual snapshots have been generated / updated for any UI changes
- [x] All developer debugging and non-functional logging has been removed
- [x] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

### Marquee effect

https://github.com/primer/brand/assets/13340707/7461cc88-6a73-41b5-ac36-252d2ce289d1

### Stacked logos

<table>

<tr>
<td valign="top">

light mode

 </td>
<td valign="top">

dark mode

</td>
</tr>
</table>

### CODE REVIEW

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+The logo suite comes with start and center align options. Only the heading and description will have visible changes to their alignment as logos will be consistently center aligned to create balance. By default, the logo suite is aligned to the start of the page.
+
+## Accessibility
+
+- Always ensure that a heading is provided to the logo suite. Headings provide leading context to the logos shown, and serve as a navigation hook for assistive technologies. It can be visually hidden if needed.
+
+## Related components
+
+- [Image](/components/Image): to display a graphical representation.
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:152]:
```suggestion
- [Image](/components/Image): to display a graphical representation.
- [Ordered List](/components/OrderedList): Use the ordered list component to display a list of numbered items.
- [Unordered List](/components/UnorderedList): Use the unordered list component to present a collection of items without specific ordering.
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+### Logo bar appearance
+
+The logo suite offers three appearance options:
+
+- `muted`: The muted option is suitable for displaying 6+ logos and displays logos in a `secondary` color. It allows for two rows of logos, maintaining a clean and organized appearance.
+- `emphasis`: The emphasis option is designed for spotlighting ~5 logos in the `primary` color. It displays logos larger, drawing more attention to the logos.
+- `marquee`: The marquee option provides a flowing logo animation and can accommodate up to 12 logos. It is ideal for scenarios with limited page real estate.
+
+The muted and emphasis options on the logo bar contains an optional divider. This divider can serve as a visual separator. These three options can be used alongside one another when logo suite components are used adjacently.
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:110]:
```suggestion
All three options on the logo bar contains an optional divider. This divider can serve as a visual separator. These three options can be used adjacently.
```

**Review by jesskuo4 — COMMENTED** [review]
Some small comments / questions, I'll be sure to edit the index once some specs are finalized :)

**Comment by jesskuo4** [comment]
Hi Reza! Thank you for engineering this so quickly, this is amazing ✨ 

One question I have about the divider is the length it spans and how we currently constrain the length. It seems like there are two options for us to consider,:
- The first is the span the divider to the container size. 
    - but this might create odd visual affects if the majority of the logos are centered in the middle of the page
- The second option is to span the divider to the length of the logobar. 
    - this might create visual inconsistencies especially if the second bar is longer than the first or we have a marquee below it. 
    
I was wondering if it's possible to test this component in the preview of the universe page? That may probably help with seeing the container of this component in an example? What do you think?

image
image

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+Use the logo suite component to showcase a list of logos, such as for clients, sponsors, or related enterprises. Use the logo suite to build credibility, trust, and provide social proof by displaying consistent logos.
+
+Logo suite should be used for groups of logos more than 4 and less than 12. In scenarios hoping to showcase more than 12 logos, considering stacking two components together. To create the best visual consistency and balance, rearrange and resize the logos when necessary to create similar area distribution and attention to each of the logos.
+
+### Arrangement
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/a2b98ef8-00d4-4a19-93cd-3c746fd40929"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:30]:
```suggestion
      src="https://github.com/primer/brand/assets/131988618/da2bd3f3-c7d8-4880-b30d-c05dd4897ba6"
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+      alt=""
+    />
+    <Caption>
+      Arrange the logos so that the sizes, heights, colors, and areas of the
+      separate logos create consistency and balance.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/327b2db7-d337-4788-b0f3-e8c5f80c7af7"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:40]:
```suggestion
      src="https://github.com/primer/brand/assets/131988618/99c9fbad-5540-43b7-93b7-00ac5c22c868)f7"
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+      alt=""
+    />
+    <Caption>
+      Don’t place logos without considering the spatial balance of the logo
+      suite. Arrange to prevent using more than two rows.
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:45]:
```suggestion
      suite. Rearrange the logos to prevent using more than two rows or imbalance. 
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Contextual information
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/4c958008-73bb-4739-81fb-adbbc4541cff"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:55]:
```suggestion
      src="https://github.com/primer/brand/assets/131988618/79680b75-9e3b-4eb1-9371-2d8ee45a6dc1"
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+      alt=""
+    />
+    <Caption>
+      Do publish the logo suite with context. Introduce the logos and, when
+      applicable, share how the logos have been grouped together.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/7099512c-c0ae-4187-ae4e-7214790d5e18"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:65]:
```suggestion
      src="https://github.com/primer/brand/assets/131988618/5ab14ace-9c9e-425f-8b5c-73de82340733"
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Color uniformity
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/3245bcd5-4304-4d84-872e-6d30b30427d6"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:80]:
```suggestion
      src="https://github.com/primer/brand/assets/131988618/5dca9fa3-856b-46f0-adeb-ecaec65e8b96"
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+      alt=""
+    />
+    <Caption>
+      Ensure that the colors are consistent across the logos to match the logo
+      bar option. Ensure spacing between logos are consistent.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/a81f3423-b708-45c8-affc-d3d45a4b7cd8"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:90]:
```suggestion
      src="https://github.com/primer/brand/assets/131988618/8912e356-421a-4c5b-9d54-27d5bdaeb8ba"

```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+    2 rows (when using 6-12 logos).
+  </Caption>
+</div>
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/d2dd578c-fce8-4999-ae07-066a616653ac"
+    alt=""
+  />
+  <Caption>The marquee options allows for different speed options.</Caption>
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:139]:
```suggestion
  <Caption>The marquee options allows for different speed options. Do not fill more than 12 logos.</Caption>
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+  />
+  <Caption>
+    The muted option is the default logo bar. It is the best option for creating
+    2 rows (when using 6-12 logos).
+  </Caption>
+</div>
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/d2dd578c-fce8-4999-ae07-066a616653ac"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:136]:
```suggestion
    src="https://github.com/primer/brand/assets/131988618/409d7a90-d086-438b-80ab-31350f6c42fa"

```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+  />
+  <Caption>
+    The muted option is the default logo bar. It is the best option for creating
+    2 rows (when using 6-12 logos).
+  </Caption>
+</div>
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/6f239722-cb7b-4976-8782-09afbb1879b7"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:125]:
```suggestion
    src="https://github.com/primer/brand/assets/131988618/ce3e22f1-9852-4869-830b-5d54b6a1a039"

```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+    alt=""
+  />
+  <Caption>
+    The muted option is the default logo bar. It is the best option for creating
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:129]:
```suggestion
    The emphasis option can be used to spotlight logos. It is best used with 6 or less logos to create 1 row. 
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+    2 rows (when using 6-12 logos).
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:130]:
```suggestion
```

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 15]
```diff
@@ -0,0 +1,152 @@
...
+
+- `muted`: The muted option is suitable for displaying 6+ logos and displays logos in a `secondary` color. It allows for two rows of logos, maintaining a clean and organized appearance.
+- `emphasis`: The emphasis option is designed for spotlighting ~5 logos in the `primary` color. It displays logos larger, drawing more attention to the logos.
+- `marquee`: The marquee option provides a flowing logo animation and can accommodate up to 12 logos. It is ideal for scenarios with limited page real estate.
+
+The muted and emphasis options on the logo bar contains an optional divider. This divider can serve as a visual separator. These three options can be used alongside one another when logo suite components are used adjacently.
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/5ad51788-25a4-401a-91b0-02ec7407aa31"
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:114]:
```suggestion
    src="https://github.com/primer/brand/assets/131988618/d525e6b1-bf67-4684-9300-401ef7686b47"

```

**Review by jesskuo4 — COMMENTED** [review]
Editing the index file for reference images with msft-owned logos. Changed some small descriptions but please feel free to change accordingly or not 👍

File: packages/react/src/LogoSuite/LogoSuite.stories.tsx [hunk 16]
```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import {Meta} from '@storybook/react'
+import {LogoSuite} from './LogoSuite'
+
+export default {
```
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.stories.tsx:6]: 👍 done

Screenshot 2023-08-02 at 10 54 03

File: packages/react/src/LogoSuite/LogoSuite.visual.spec.ts-snapshots/Visual-Comparison-LogoSuite-LogoSuite-Align-Justified-1-linux.png
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.visual.spec.ts-snapshots/Visual-Comparison-LogoSuite-LogoSuite-Align-Justified-1-linux.png:1]: Yeah, thanks for bringing that up. I looked at other designs like Universe and noticed that their implementation of `justified` is only applicable to logos... the heading stays centralised. Happy to `align-start` the heading and description if you prefer though. AFAIK your design specs don't include a `justified` option, so I only added this in for sites like Universe.

File: packages/react/src/LogoSuite/LogoSuite.test.tsx [hunk 17]
```diff
@@ -0,0 +1,246 @@
...
+        <LogoSuite.Logobar data-testid={testId}>
+          <svg data-testid="svg1" />
+          <svg data-testid="svg2" />
+        </LogoSuite.Logobar>
+      </LogoSuite>,
+    )
+
+    expect(getByText(mockHeading)).toBeInTheDocument()
+    expect(getByText(mockDescription)).toBeInTheDocument()
+    expect(getByTestId(testId).children.length).toBe(2)
```
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.test.tsx:34]: > are setting the minimum logo requirements to be 2?

No, this is just a unit test assertion checking that the two logos i'm trying to render do in fact appear correctly, no more, no less. 

There's currently no logic around how many logos you can have in LogoSuite. Do we need to set that on our end?

File: packages/react/src/LogoSuite/LogoSuite.features.stories.tsx [hunk 18]
```diff
@@ -0,0 +1,359 @@
...
+    <LogoSuite hasDivider={false}>
+      <LogoSuite.Heading visuallyHidden>Our customers</LogoSuite.Heading>
+      <LogoSuite.Logobar marquee marqueeSpeed="slow">
+        {logos.slice(0, 4)}
+      </LogoSuite.Logobar>
+    </LogoSuite>
+  </>
+)
+
+export const WithLinks = () => (
```
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.features.stories.tsx:348]: Thanks for bringing this up 🙇. Great question. Ideally we wouldn't want to allow linking for the reasons I mentioned. However, I've noticed a conversation happening [here](https://github.slack.com/archives/C051ULZRKDE/p1690501647462559), where they are thinking of using links in the logo area. I've pre-emptively ensuring that links work, which is why the story exists but I don't reference links in interface guidelines or react docs because it's not a recommended pattern. Does that make sense. Also happy to remove the story if you'd prefer.

File: packages/react/src/LogoSuite/LogoSuite.features.stories.tsx [hunk 19]
```diff
@@ -0,0 +1,359 @@
...
+        <LogoSuite.Description>Body text maximus ligula felis, non egestas dolor rutrum vel.</LogoSuite.Description>
+        <LogoSuite.Logobar variant="emphasis">
+          <React.Fragment>{logos.slice(0, 4)}</React.Fragment>
+        </LogoSuite.Logobar>
+      </LogoSuite>
+    </Grid.Column>
+  </Grid>
+)
+
+export const Stacked = () => (
```
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.features.stories.tsx:230]: 👍 that's done

File: apps/docs/content/components/LogoSuite/index.mdx (hunk 1 above)
Comment (rezrah) [reply apps/docs/content/components/LogoSuite/index.mdx:152]: 👍 done

(I excluded those originally as I wasn't sure how relevant they were. No strong opinions on it though, so added them in)

File: apps/docs/content/components/LogoSuite/index.mdx (hunk 14 above)
Comment (rezrah) [reply apps/docs/content/components/LogoSuite/index.mdx:130]: @jesskuo4 - is this suggestion correct?

**Comment by rezrah** [comment]
> I was wondering if it's possible to test this component in the preview of the universe page? That may probably help with seeing the container of this component in an example? What do you think?

@jesskuo4 - there's a PR open for this now, with a preview url. I'll DM you the deets.

File: packages/react/src/LogoSuite/LogoSuite.features.stories.tsx (hunk 18 above)
[the remark this answers is not in the document]
Comment (jesskuo4) [reply packages/react/src/LogoSuite/LogoSuite.features.stories.tsx:348]: Hi Reza! Thanks so much, no worries at all, I think having a link option is great to future proof it and as long as usage is consistent/accessible, I don't think there will be a huge issue there. Since we are prioritizing for Universe, we can definitely work according to their current needs & add a link option :) 

Thanks for sharing this thread, two things I want to bring up briefly: 
> Please arrange in hierarchy of sponsorship level. It should be Marquee largest and top row (3 max) // then Principal + Supporting next size (10-15 total) and second row // then Community smallest and lower row.
This was mentioned in the thread which is really interesting to me. In my head, I was thinking the marquee would be the bottom row as the community and least spotlight sponsors. And emphasis would be the largest top row. What is your opinion on that? Should we follow up in the thread to confirm if they are sure that the spotlight sponsors should be in an animation? 

> Also I think we can make the hierarchy less extreme for logo size. Marquee = 100%. Principal/Supporting = 85%. Community = 70%
This was also mentioned in the thread 🤔 regardless of what the top row is (marquee or emphasis), it seems that they might want it a bit smaller. What are your thoughts on this? If you agree, we can make the top row smaller as that seems like what they want. (I took the original measurements from the figma draft of universe)

File: packages/react/src/LogoSuite/LogoSuite.test.tsx (hunk 17 above)
[the remark this answers is not in the document]
Comment (jesskuo4) [reply packages/react/src/LogoSuite/LogoSuite.test.tsx:34]: Thank you for explaining this and sharing with me how the code breaks down, I really appreciate it! I agree, I don't think we need test cases since I can see some instances where designers or engineers in the future find more fit to use more rows (if we have more sponsors/logos). A recommendation for interface guidelines I think will suffice 👍 thank you!

File: apps/docs/content/components/LogoSuite/index.mdx (hunk 1 above)
Comment (jesskuo4) [reply apps/docs/content/components/LogoSuite/index.mdx:153]: Sounds good! Thank you 😄 Actually Josep mentioned to add those lists, I also don't have a strong opinion and wasn't sure how relevant, thank you for adding 💯

File: apps/docs/content/components/LogoSuite/index.mdx (hunk 14 above)
Comment (jesskuo4) [reply apps/docs/content/components/LogoSuite/index.mdx]: Hi Reza! Thanks for pointing this out, I don't think I explained this clearly in the caption, let me try again. Let me know if you agree / think this updated version is clear. 

I want to say that out of the three options, the muted option is best for logos that appear in 2 rows.

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 20]
```diff
@@ -0,0 +1,152 @@
...
+    alt=""
+  />
+  <Caption>
+    The muted option is the default logo bar. It is the best option for creating
```
Comment (jesskuo4) [inline apps/docs/content/components/LogoSuite/index.mdx:119]:
```suggestion
    The muted option is the default logo bar. To display 6-12 logos in 2 rows, it is preferred to use the muted option instead of emphasis or marquee. 
```

**Review by jesskuo4 — COMMENTED** [review]
Changed some of the suggestions to be more clear/responded to comments, thank youu

File: packages/react/src/LogoSuite/LogoSuite.features.stories.tsx (hunk 18 above)
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.features.stories.tsx:348]: > What is your opinion on that? Should we follow up in the thread to confirm if they are sure that the spotlight sponsors should be in an animation?

I agree we should address respond to that thread. Before that, could you please check in with @ajashams and @nsolerieu to confirm that our version is correct and aligns with their expectations for it please? Perhaps they could also support the conversation around this as we try to discourage ad-hoc design changes that don't go through site design review.

File: .changeset/tall-penguins-walk.md [hunk 21]
```diff
@@ -0,0 +1,15 @@
+---
+'@primer/brand-primitives': minor
+---
+
+Added design tokens for the logo suite component
+
+```css
+--brand-LogoSuite-color-logo-muted
+--brand-LogoSuite-color-logo-emphasis
```
[the remark this answers is not in the document]
Comment (langermank) [reply .changeset/tall-penguins-walk.md:9]: I'll add my thoughts on this as well 😄 

First, @rezrah it looks like you aren't actually referencing these tokens in the [CSS](https://github.com/primer/brand/pull/362/files#diff-cafdf90ef9bfe8c8a3ee14ec629478a737f91a87829873a75777d85bb2ecd752R56). Is that intentional? 

IMO it makes more sense to just use the colors directly as you have in the CSS. Its just less overhead to manage, and if we decide to add more variants in the future we can always add component specific tokens at that time (easier to add than to remove).

[the remark this answers is not in the document]
Comment (jesskuo4) [reply .changeset/tall-penguins-walk.md:9]: Hi Reza! Thanks for mentioning this! I'm currently doing some work updating primer token names (specifically color) and some of the work with that includes simplifying or removing tokens that can potentially be replaced with some of our base tokens. Would love to hear your opinion on this 👍

[the remark this answers is not in the document]
Comment (rezrah) [reply .changeset/tall-penguins-walk.md:9]: > First, @rezrah it looks like you aren't actually referencing these tokens in the [CSS](https://github.com/primer/brand/pull/362/files#diff-cafdf90ef9bfe8c8a3ee14ec629478a737f91a87829873a75777d85bb2ecd752R56). Is that intentional?

Oh wow 🤦. Thanks for pointing this out, must have forgotten to update those values. This is _not_ intentional, no.

Totally appreciate the desire to reduce token footprint. In this instance I feel that providing these two color tokens is more consistent to how we approach API extensibility in Primer Brand generally (see Label for a similar example. 

> I'm currently doing some work updating primer token names (specifically color) and some of the work with that includes simplifying or removing tokens that can potentially be replaced with some of our base tokens. 

👀 oh interesting. Is that work for brand, product libraries or both? In Primer Brand, we've historically created component tokens whenever we want to provide either escape hatches or additional low-level, user configurability. So it's quite a powerful tool, which our users also utilise regularly. Interested to hear more about this work if you can share some details? Also just FYI removing tokens usually requires managing breaking changes, so we'd need to plan ahead for things like that.

[the remark this answers is not in the document]
Comment (langermank) [reply .changeset/tall-penguins-walk.md:9]: > Totally appreciate the desire to reduce token footprint. In this instance I feel that providing these two color tokens is more consistent to how we approach API extensibility in Primer Brand generally

Sure, don't consider this blocking feedback. 

> I'm currently doing some work updating primer token names (specifically color) and some of the work with that includes simplifying or removing tokens that can potentially be replaced with some of our base tokens.

@jesskuo4 is referring to the token name refactoring work that we're preparing for. So nothing new, just the same project that we've discussed in the past. In auditing the existing tokens we've found some areas that we can reduce the token footprint, but this is mostly scoped to control tokens so not necessarily a component-wide concern.

File: packages/react/src/LogoSuite/LogoSuite.visual.spec.ts-snapshots/Visual-Comparison-LogoSuite-LogoSuite-Align-Justified-1-linux.png
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.visual.spec.ts-snapshots/Visual-Comparison-LogoSuite-LogoSuite-Align-Justified-1-linux.png:1]: 👋 @jesskuo4 - 👍 this is done

**Review by jesskuo4 — APPROVED** [review]
Looks good! I'll forward to Nicolas to check if usage / interface guidelines are aligned but everything LGTM 😄

File: packages/react/src/LogoSuite/LogoSuite.features.stories.tsx (hunk 18 above)
[the remark this answers is not in the document]
Comment (rezrah) [reply packages/react/src/LogoSuite/LogoSuite.features.stories.tsx:347]: if it's just interface guidelines, we can do that in a follow up PR. Is that okay?

File: apps/docs/content/components/LogoSuite/index.mdx [hunk 22]
```diff
@@ -0,0 +1,155 @@
...
+  <Caption>The marquee options allows for different speed options.</Caption>
+</div>
+
+### Alignment
+
+The logo suite comes with start and center align options. Only the heading and description will have visible changes to their alignment as logos will be consistently center aligned to create balance. By default, the logo suite is aligned to the start of the page.
+
+## Accessibility
+
+- Always ensure that a heading is provided to the logo suite. Headings provide leading context to the logos shown, and serve as a navigation hook for assistive technologies. It can be visually hidden if needed.
```
Comment (josepmartins) [inline apps/docs/content/components/LogoSuite/index.mdx:149]:
```suggestion
Always ensure that a heading is provided to the logo suite. Headings provide leading context to the logos shown, and serve as a navigation hook for assistive technologies. It can be visually hidden if needed.
```

**Review by josepmartins — APPROVED** [review]
LGTM! 👍  left a minor suggestion

**Review by danielguillan — APPROVED** [review]

**Comment by rezrah** [comment]
Thanks @nsolerieu - @jesskuo4 perhaps something we can add to the interface guidelines in a follow up PR?

**Comment by jesskuo4** [comment]
@rezrah Yes, sounds good! I actually really wanted to include examples similar to what @nsolerieu mentioned but ultimately deleted that do/don't guideline because of the legal logo struggle (only being able to use msft-owned logo) 😭 That made it hard to replicate examples since my first trial it was hard to get similar examples as the home depot and american airlines that you mentioned. 

Let me dig deeper and see how I can replicate something similar for the guidelines
